### ISSUE TITLE
NVDA fails to select menu item in chrome

### ISSUE THREAD

**Issue by jqureshi-sans** [nvaccess/nvda#16542]
I can't seem to figure out the root cause of this menu bug when users are using NVDA and Chrome on www.sans.org. I attached the logs with a crash reported below.
This issue was not occurring on older versions of chrome so I'm leaning towards thinking this is an issue with some new chrome change but wanted to get your opinions if possible.
### Steps to reproduce:
While running NVDA navigate to www.sans.org move the selector over to the login link.
### Actual behavior:
When the user selects the login link and the selector will navigate to a different element.

### Expected behavior:
The user should be navigated over to a different page to login

### NVDA logs, crash dumps and other attachments:
IO - speech.speech.speak (14:02:04.893) - MainThread (42408):
Speaking [LangChangeCommand ('en_US'), 'Log In', 'link', 'visited', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (14:02:06.211) - winInputHook (31796):
Input: kb(desktop):enter
DEBUGWARNING - NVDAObjects.IAccessible.IA2TextTextInfo._getLineOffsets (14:02:06.307) - MainThread (42408):
IAccessibleText::textAtOffset failed
Traceback (most recent call last):
  File "NVDAObjects\IAccessible\__init__.pyc", line 349, in _getLineOffsets
  File "comtypes\_memberspec.pyc", line 482, in __call__
_ctypes.COMError: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
IO - speech.speech.speak (14:02:06.312) - MainThread (42408):
Speaking [LangChangeCommand ('en_US'), 'Search field', 'edit', CancellableSpeech (still valid), 'Search SANS...', CancellableSpeech (still valid), 'blank']
IO - inputCore.InputManager.executeGesture (14:02:07.252) - winInputHook (31796):
Input: kb(desktop):tab
IO - speech.speech.speak (14:02:07.308) - MainThread (42408):
Speaking [LangChangeCommand ('en_US'), 'SANS Sites', 'link', 'visited', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (14:02:07.419) - winInputHook (31796):
Input: kb(desktop):tab
IO - speech.speech.speak (14:02:07.460) - MainThread (42408):
Speaking [LangChangeCommand ('en_US'), 'Log In', 'link', 'visited', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (14:02:08.203) - winInputHook (31796):
Input: kb(desktop):enter
DEBUG - browseMode.BrowseModeDocumentTreeInterceptor.terminate (14:02:09.077) - MainThread (42408):
Saving caret position Offsets(startOffset=102, endOffset=102) for document at https://www.sans.org/
DEBUG - treeInterceptorHandler.killTreeInterceptor (14:02:09.078) - MainThread (42408):
Killed treeInterceptor: <NVDAObjects.IAccessible.chromium.ChromeVBuf object at 0x0D935E50>
DEBUG - NVDAObjects.IAccessible.chromium.Document._get_treeInterceptorClass (14:02:09.079) - MainThread (42408):
loadChromiumVBufOnBusyState feature flag is True, vBuf WILL be loaded when state of the document is busy.
DEBUG - treeInterceptorHandler.update (14:02:09.079) - MainThread (42408):
Adding new treeInterceptor to runningTable: <NVDAObjects.IAccessible.chromium.ChromeVBuf object at 0x0D953730>
DEBUG - virtualBuffers.VirtualBuffer._loadBuffer (14:02:09.081) - NVDAObjects.IAccessible.chromium.VirtualBuffer.loadBuffer (11044):
Buffer load took 0.001 sec, 70 chars

#### NVDA installed/portable/running from source:
Portable
#### NVDA version:
2024.1
#### Windows version:
Windows 10
#### Does the issue still occur after restarting your computer?
Did not test but the version of NVDA I used was not installed on the computer
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
No NVDA addons were used

**Comment by seanbudd** [comment]
Welcome @jqureshi-sans ,

We are unable to process this issue as it stands.

This is a problem with the implementation of the website.
It appears that you are the author of this website.

To be able to reproduce and understand this issue, we need a minimal code sample to ensure it is not an authoring issue. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

Kind Regards,
NV Access Software Developers
